src/table/cells/__tests__/handle-key-press.spec.js

```diff
@@ -192,4 +240,62 @@ describe('handle-key-press', () => {
       });
     });
   });
+
+  describe('headHandleKeyPress', () => {
+    let rowIndex;
+    let colIndex;
+    let evt = {};
+    let rootElement = {};
+    let changeSortOrder;
+    let layout;
+    let isDim;
+
+    beforeEach(() => {
+      rowIndex = 0;
+      colIndex = 0;
+      evt = {
+        key: 'ArrowDown',
+        stopPropagation: sinon.spy(),
+        preventDefault: sinon.spy(),
+        target: {
+          blur: sinon.spy(),
+          setAttribute: sinon.spy(),
+        },
+      };
+      rootElement = {
+        getElementsByClassName: () => [{ getElementsByClassName: () => [{ focus: () => {}, setAttribute: () => {} }] }],
+      };
+      changeSortOrder = sinon.spy();
+    });
+
+    describe('ArrowDown', () => {
```

Comment:
No reason for a sepparate describe for every case, just add it to de `it` description

Reply:
Fixed. Thanks